x/superfluid/keeper/slash.go

```diff
@@ -109,6 +109,9 @@ func (k Keeper) prepareConcentratedLockForSlash(ctx sdk.Context, lock *lockuptyp
 	if err != nil {
 		return sdk.AccAddress{}, sdk.Coins{}, err
 	}
+
+	// Q: this would make the amounts calculated in alculateUnderlyingAssetsFromPosition
+	// be rounded down. Is this fine? I think yes, should be minimal impact.
 	slashAmtNeg := slashAmt.Neg()
```

Comment:
Note: would like to discuss this as well

Reply:
To clarify, I think your concern is that, since CalcUnderlyingAssetsFromPositions rounds the assets up, the slashAmt does not take this into consideration right?

This does not feel problematic to me, but would be good to note in the code comments.